.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
```

Comment:
deze gitignore staat te diep in het project. Zou je ook de .idea map aan de juist gitignore toe willen voegen?

Reply:
gewoon /.idea toevoegen?

Reply:
Je moet er dan ook voor zorgen dat git stopt met het volgen van die map, want dat doet hij niet als de map al een keer gecommit is, ook al staat hij in de gitignore. Dat kan met `git rm --cached /.idea` . En dan opnieuw committen, dan staat /.idea als het goed is ook niet meer op github.